### PR TITLE
[ADD] invoice_comment_template: add template to partner

### PR DESCRIPTION
**Pull request by cubells**
Add missing feature from older version: now you can assign a comment template to partner.

- Update spanish translation

cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
invoice_comment_template? Why don't we called it `account_invoice_comment_template`?

File: invoice_comment_template/models/account_invoice.py [hunk 1]
```diff
@@ -27,3 +27,13 @@ def _set_note2(self):
         comment = self.comment_template2_id
         if comment:
             self.note2 = comment.get_value(self.partner_id.id)
+
+    @api.onchange('partner_id', 'company_id')
+    def _onchange_partner_id(self):
+        res = super(AccountInvoice, self)._onchange_partner_id()
+        comment_template = self.partner_id.comment_template_id
+        if comment_template.position == 'before_lines':
```
Comment (pedrobaeza) [inline invoice_comment_template/models/account_invoice.py:36]: Don't perform anything if not `comment_template`

File: invoice_comment_template/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# Copyright 2018 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    comment_template_id = fields.Many2one(
```
Comment (pedrobaeza) [inline invoice_comment_template/models/res_partner.py:11]: Add this field to commercial fields.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by cubells** [comment]
invoice_comment_template is the original module name @pedrobaeza 
Do you think I must change it?

**Comment by pedrobaeza** [comment]
Well, not now. We should do it before, but now it's late. On next version...

File: invoice_comment_template/models/res_partner.py (hunk 2 above)
Comment (cubells) [reply invoice_comment_template/models/res_partner.py:11]: added.

File: invoice_comment_template/models/account_invoice.py (hunk 1 above)
Comment (cubells) [reply invoice_comment_template/models/account_invoice.py:36]: It's removed template_id if there is not not comment_template

**Comment by cubells** [comment]
@pedrobaeza can you recheck?

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**